src/charmed_kubeflow_chisme/testing/README.md

```diff
@@ -0,0 +1,62 @@
+# Testing
+
+Tools for unit or integration testing, such as importable and reusable tests.
+
+# Contents
+
+## `test_leadership_events`
```

Comment:
This readme has good content.  It is exactly what I'd expect in a readme, but also feels like something I should see in the code's docstrings as well (that's a likely place for people to look).  But, it is a pain to maintain them in both places... not sure what best to do about that

Reply:
I used [lazydocs](https://github.com/ml-tooling/lazydocs) for docstring -> markdown conversion.
To try it out:
`pip install lazydocs`
In a directory containing the file with docstring to be converted:
`lazydocs <filename> --output-path stdout > README.md`